File: code/77200.pas
program Bai200;
  var n: integer;
      x: integer;
  begin
  readln(n);
    if n < 2 then 
      begin
      write('NO ');
      exit;
      end;
  for x := 2 to n - 1 do 
     if n mod x = 0 then 
      begin
      write('NO');
      exit;
      end;
    write('YES ');
end.

File: code/77200.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,x;
    cin>>n;
    if (n<2){
        cout<<"NO";
        return 0;
    }
    for (x=2; x<=n-1; x++)
        if (n%x==0){
            cout<<"NO";
            return 0;
        }
    cout<<"YES";
    return 0;
}